libraries/AP_RCTelemetry/AP_CRSF_Telem.cpp

```diff
@@ -381,10 +381,10 @@ void AP_CRSF_Telem::calc_attitude()
     AP_AHRS &_ahrs = AP::ahrs();
     WITH_SEMAPHORE(_ahrs.get_semaphore());
 
-    _telem.bcast.attitude.roll_angle =  htobe16(roundf(_ahrs.roll * 10000.0f));
-    _telem.bcast.attitude.pitch_angle =  htobe16(roundf(_ahrs.pitch * 10000.0f));
-    _telem.bcast.attitude.yaw_angle =  htobe16(roundf(_ahrs.yaw * 10000.0f));
-
+    _telem.bcast.attitude.roll_angle =  htobe16(roundf(wrap_2PI(_ahrs.roll) * 10000.0f));
+    _telem.bcast.attitude.pitch_angle =  htobe16(roundf(wrap_2PI(-_ahrs.pitch) * 10000.0f));
+    _telem.bcast.attitude.yaw_angle =  htobe16(roundf(wrap_2PI(_ahrs.yaw) * 10000.0f));
+    
```

Comment:
You should kill this whitespace

Reply:
whitespace fixed. negative pitch required to align with existent conventions (BF). seems congruent with -ahrs.pitch use in OSD horizon.